src/thunderbots/software/util/canvas_messenger/canvas_messenger.h

```diff
@@ -0,0 +1,84 @@
+#pragma once
+
+#include <ros/ros.h>
+
+#include <map>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "thunderbots_msgs/CanvasLayer.h"
+
+// Forward declaration
+namespace ros
+{
+    class NodeHandle;
+    class Publisher;
+}  // namespace ros
```

Comment:
I don't think you should need this if you include `<ros/ros.h>`?

Reply:
Yup, removed